### PR TITLE
Update docs describing esmda restarts

### PR DESCRIPTION
**Pull request by dafeda**
## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Updated documentation
- [ ] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: docs/getting_started/howto/esmda_restart.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Restarting ES-MDA
+-----------------
+Imagine you are conducting an experiment consisting of three iterations of ES-MDA, 
+with each iteration potentially taking hours or even days.
+For the purposes of this example, assume that the "Relative weights" are set to ``4, 2, 1``.
+
+As part of running this experiment, ERT creates four ensembles, which by default are named ``default_0``, ``default_1`` and ``default_2``, ``default_3``.
+Sadly, too many realizations fail during the model evaluation in ``default_3``, leaving no responses to analyse.
+Such failures can arise for various reasons, such as licensing problems with the simulator, unphysical parameterization etc.
```
Comment (oyvindeide) [inline docs/getting_started/howto/esmda_restart.rst:9]: But if the parameterization is unphysical, a restart will not help, so perhaps add some information about which cases are worth restarting from?

Comment (dafeda) [reply docs/getting_started/howto/esmda_restart.rst]: Thanks 👍 
I have also updated the sentence that started with "Sadly".

File: docs/getting_started/howto/esmda_restart.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Restarting ES-MDA
+-----------------
+Imagine you are conducting an experiment consisting of three iterations of ES-MDA, 
+with each iteration potentially taking hours or even days.
+For the purposes of this example, assume that the "Relative weights" are set to ``4, 2, 1``.
+
+As part of running this experiment, ERT creates four ensembles, which by default are named ``default_0``, ``default_1`` and ``default_2``, ``default_3``.
+It is possible that too many realizations fail during the model evaluation in ``default_3``, leaving no responses to analyse.
+Various factors, including licensing issues with the simulator, can lead to such failures.
```
Comment (oyvindeide) [inline docs/getting_started/howto/esmda_restart.rst:9]:
```suggestion
Various factors, for example license server issues or compute cluster downtime, can lead to such failures.
```

**Review by oyvindeide — APPROVED** [review]
LGTM! Just a small suggestion

**Pull request merged**